mods/persistence/modules/mob/malnourished.dm

```diff
@@ -0,0 +1,28 @@
+/mob/living/carbon/human/handle_exertion()
+	if((!nutrition || !hydration) && (!InStasis()))
+		add_chemical_effect(CE_TOXIN, 1)
+		adjustToxLoss(DEFAULT_HUNGER_FACTOR/1000)
+
+	for(var/obj/item/organ/internal/stomach/stomach in src)
+		if((!nutrition) && (!InStasis()))
+			stomach.take_internal_damage(DEFAULT_HUNGER_FACTOR/1000)
+
+	for(var/obj/item/organ/internal/kidneys/kidneys in src)
+		if((!hydration) && (!InStasis()))
+			kidneys.take_internal_damage(DEFAULT_HUNGER_FACTOR/1000)
+	. = ..()
+
+/mob/living/carbon/human/handle_regular_status_updates()
+	if((!nutrition || !hydration) && (!InStasis()))
+		add_chemical_effect(CE_TOXIN, 1)
+		adjustToxLoss(0.5001)
```

Comment:
Why does this have the trailing .0001?

Reply:
an attempt to heal damage is constantly made, this .0001 allows slow damage to the liver. If the person is not in stasis they will slowly build up toxin and liver damage, which will take real days to do considerable damage. The heal attempt heals .5 per tick. the .0001 is the difference.